src/components/popover/popover.tsx

```diff
@@ -39,6 +39,16 @@ import { portalContains } from '../portal/contains';
  * When a popover is displayed, interactions with other controls are blocked,
  * until user dismisses the popover.
  *
+ * :::warning
+ * In order for the popover to pop on the correct side of the element that
+ * triggers it, you need to wrap both of them in a container and make sure that
+ * the container:
+ * 1. is visually as big as the element that triggers the popover
```

Comment:
Can't we provide the container ourselves, and put the trigger in a slot. I'm pretty sure that's what we do in limel-menu, and that works just fine…

Then we wouldn't need this big warning.

Reply:
We also thought about that. So maybe we should try to do that instead 🙂

Reply:
I don't know how or where we should do this :/ can you suggest a solution in code?

Reply:
I can try to do it 🙂

Reply:
Give me a holler tomorrow if you don't manage, but if you look at limel-menu, I think it'll be pretty straightforward 👍